tasks/base_tasks.py

```diff
@@ -986,12 +990,88 @@ def check_universe_in_aggregations(self):
                              "but lack of 'universe' target: {columns}".format(
                                 table=self.output().table, columns=', '.join([x[0] for x in result])))
 
+    def check_table_to_table_relations(self):
+        session = current_session()
+        # Relations between tables defined as follows:
+        # TABLE(SOURCE) -- COLUMN_TABLE(source) -- COLUMN_TO_COLUMN -- COLUMN_TABLE(target) -- TABLE(target)
+        # that are not in OBS_TABLE_TO_TABLE
+        query = ('''
+                 SELECT DISTINCT ts.id || '.' || cs.id ||
+                 ' / ' || tt.id || '.' || ct.id || ' (' || cc.reltype || ')' relation
+                 FROM observatory.obs_table ts, observatory.obs_column_table cts, observatory.obs_column cs,
+                 observatory.obs_column_to_column cc,
+                 observatory.obs_table tt, observatory.obs_column_table ctt, observatory.obs_column ct
+                 WHERE cc.reltype = 'geom_ref'  -- WARNING! only geom_ref reltype ATM
+                 AND ts.tablename = '{table}'
```

Comment:
One question, why only the source and not both?

Reply:
We are checking that the table that we are currently working on (the source) doesn't lack relations with any other table (the targets)